models/Post.php

```diff
@@ -2,23 +2,27 @@
 
 use Db;
 use Url;
-use App;
-use Str;
 use Html;
 use Lang;
 use Model;
 use Markdown;
 use BackendAuth;
-use ValidationException;
-use RainLab\Blog\Classes\TagProcessor;
-use Backend\Models\User;
 use Carbon\Carbon;
-use Cms\Classes\Page as CmsPage;
 use Cms\Classes\Theme;
+use Backend\Models\User;
+use ValidationException;
+use Cms\Classes\Controller;
+use Cms\Classes\Page as CmsPage;
+use RainLab\Blog\Classes\TagProcessor;
+use October\Rain\Database\Traits\Validation;
 
+/**
+ * Class Post
+ */
 class Post extends Model
 {
-    use \October\Rain\Database\Traits\Validation;
```

Comment:
Leave the above style please, it's used everywhere else in the code base

Reply:
Done